src/App.js

```diff
@@ -38,7 +38,11 @@ function App() {
       </header>
 
       {events.map((event) => (
-        <Card title={event.title} subTitle={event.date.start}>
+        <Card 
+          title={event.title} 
+          subTitle={event.date.start}
+          key={event.description}
```

Comment:
description might not be unique. You can access the index as the second parameter of the map function and add the key 
```jsx
key={`${index}-${event.description}`}
```

Reply:
Is this PR still relevant? I cannot see the card component anywhere now.

Reply:
The default branch `main` has moved forwards, please update your `main` and merge it into your branch (the PR will update automatically)